NRedisTimeSeries/TimeSeriesClient.cs

```diff
@@ -275,9 +280,10 @@ public static IReadOnlyList<(string key, IReadOnlyList<TimeSeriesLabel> labels,
             (string, TsReduce)? groupbyTuple = null,
             IReadOnlyCollection<TimeStamp> filterByTs = null,
             (long, long)? filterByValue = null,
-            IReadOnlyCollection<string> selectLabels = null)
+            IReadOnlyCollection<string> selectLabels = null,
+            TimeStamp align = null)
```

Comment:
@AvitalFineRedis dont we need this to be `TimeStamp? align = null` so that TimeStamp is nullable?

Reply:
`TimeStamp` is already nullable, so we don't need the question mark.
You can see that in L98